docs/migration-guides/0.8-0.9.md

```diff
@@ -0,0 +1,37 @@
+
+> **TIP:** For more details, read release details in [GitHub](https://github.com/kyma-project/kyma/releases/tag/0.9.0) and on [Kyma Blog](https://kyma-project.io/blog/).
+
+The Headless CMS influences the services specifications persistence of the Application Connector. As a result, Kyma users who upgrade from 0.8.0 to 0.9.0 are affected, because the old location of the specifications is not supported by the Application Connector.
+To make migration as seamless as possible we introduce an automated migration script that does two things:
+- removes old and registers new custom resources for the Headless CMS and the Asset Store components
+- migrates specifications registered with the Application Connector prior to 0.9.0 release into Headless CMS
+> **TIP:** For more details, read the migration [script](https://github.com/kyma-project/kyma/blob/master/resources/cms/templates/_helper_migrate_to_cms.txt).
+
+### Migration script limitations
+
+The migration script does not address all possible use cases. Its scope focuses only on the use cases known as used by Kyma users.
+Basing on the known use cases, the script assumptions are:
+- One service has only one specification. There are no services with two different specs, such as OpenAPI and AsyncAPI.
+- Documentation registration exposed through the Metadata Service is not used in current integrations.
+
+The script takes specs from registered applications, for example from `https://minio.{$DOMAIN}/content/service-class/{$SERVICE_CLASS_ID}/{$SPEC_NAME}.json` (where spec name is `asyncApiSpec` or `apiSpec`),
+and saves them as `ClusterDocsTopic` custom resource.
+
+### Manual migration
+
+Uncovered use cases require manual migration after a successful upgrade. Read the Headless CMS documentation to understand how to use it.
+Keep in mind that prior to 0.9.0 release, documentation was kept in a single `content.json` file. Now, you can deliver content from separate `content.json` files in a package.
+
+In case of any issues with the migration, contact us on [Slack](http://slack.kyma-project.io) on the `core-and-supporting` channel.
+
+### Recommendations 
+
+We recommend to not register additional static documentation during application registration. This means we do not recommend to provide any data in `documentation` attribute in payload sent to `/services` endpoint from the Metadata service. We work now on a proposal to redesign the documentation registration payload (not specifications).
+For the time being, if you will use `documentation` attribute, and send content, the Metadata service creates a `ClusterDocsTopic` resource. Type of the source specified in the resource is set to `docs`. This type is not recognized by the Console UI. We implmeneted this temporary solution to not introduce any braking changes in current solution. Read the Headless CMS documentation to understand what `ClusterDocsTopic` resource is and what type of sources are supported.
```

Comment:
```suggestion
For the time being, if you use the `documentation` attribute and send content, the Metadata Service creates a `ClusterDocsTopic` resource. Type of the source specified in the resource is set to `docs`. This type is not recognized by the Console UI. We implemented this temporary solution to avoid introducing any breaking changes. Read the Headless CMS documentation to understand what `ClusterDocsTopic` resource is and what type of sources are supported.
```